test/test_operations.py

```diff
@@ -853,49 +853,40 @@ def __init__(self, x, y):
     nested = Nested(b, c)
     self.assertRaises(RuntimeError, lambda: xm.check_view_sharing(nested))
 
-    tmpfile = tempfile.mktemp()
-    self.assertRaises(RuntimeError, lambda: torch.save([b, c], tmpfile))
+    with tempfile.TemporaryFile() as tf:
```

Comment:
I am assuming that `torch.save()` takes file pointers on top of file paths, right?

Reply:
Yea it takes both ` a file-like object (has to implement write and flush) or a string containing a file name`.